nscplugin/src/main/scala/scala/scalanative/nscplugin/NirGenName.scala

```diff
@@ -52,12 +52,18 @@ trait NirGenName { self: NirGenPhase =>
   def genFieldName(sym: Symbol): nir.Global = {
     val owner = genTypeName(sym.owner)
     val id    = nativeIdOf(sym)
+    val scope = {
+      /* Variables are internally private, but with public setter/getter.
+       Removing this check would cause problems with reachability  */
```

Comment:
`/* ... */` comments should repeat the stars at the beginning of the lines, like this:
```suggestion
      /* Variables are internally private, but with public setter/getter.
       * Removing this check would cause problems with reachability
       */
```